public/css/style.css

```diff
@@ -412,8 +449,8 @@ h6 {
 }
 
 #hero .btn-get-started:hover {
-  background: #084933;
-  border: 2px solid #084933;
+  background: #0f2719;
+  border: 2px solid #0f2719;
 }
 
 @media (min-width: 1024px) {
```

Comment:
I can't see a lot of Mobile-specific styling here
Is Mobile Responsiveness been considered?
Also, add screen to your media query

Reply:
Okay, And Yes It is mobile friendly